src/ff8_opengl.cpp

```diff
@@ -1267,6 +1481,49 @@ void ff8_init_hooks(struct game_obj *_game_object)
 	// All possible message and ask windows
 	ff8_opcode_old_battle = (int (*)(int))ff8_externals.opcode_battle;
 	patch_code_dword((uint32_t)&common_externals.execute_opcode_table[0x69], (DWORD)&ff8_opcode_battle);
+
+	//###############################
+	// steam achievement unlock calls
+	//###############################
+	if(steam_achievements_debug_mode)
```

Comment:
Is this flag the right one being used here? Or did you mean `enable_steam_achievements`?